keyboards/gmmk/numpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2021 Glorious, LLC <salman@pcgamingrace.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "analog.h"
+
+
+// clang-format off
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+//      NUM      /       *       -
+//      7        8       9       +
+//      4        5       6       CALC
+//      1        2       3       RET
+//      0                        .
+
+  [0] = LAYOUT(
+    MO(1),   KC_PSLS,   KC_PAST,   KC_PMNS,
+    KC_P7,   KC_P8,     KC_P9,     KC_PPLS,
+    KC_P4,   KC_P5,     KC_P6,     KC_CALC,
+    KC_P1,   KC_P2,     KC_P3,     KC_PENT,
+    KC_P0,                         KC_PDOT
+  ),
+  [1] = LAYOUT(
+    _______,   KC_PSLS,   KC_PAST,   KC_PMNS,
+    KC_P7,     RGB_VAI,   KC_P9,     KC_PPLS,
+    RGB_RMOD,  KC_P5,     RGB_MOD,   KC_CALC,
+    KC_P1,     RGB_VAD,   KC_P3,     KC_PENT,
+    RGB_TOG,                         QK_BOOT
+  )
+};
+// clang-format on
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (!clockwise) {
+      tap_code(KC_VOLU);
+    } else {
+      tap_code(KC_VOLD);
+    }
+    return true;
+}
+#endif
```

Comment:
```suggestion

```
This isn't needed as this already exists in [`numpad.c`](https://github.com/qmk/qmk_firmware/pull/20067/files#diff-22bd728dacf18385a028533798f924c6eee6dbd7f17b2d3bd64b03711aa33af9R122-R134) at the keyboard level.